File: android/src/main/java/y2k/joyreactor/platform/ImageRequest.java
package y2k.joyreactor.platform;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import y2k.joyreactor.images.BaseImageRequest;

import java.io.File;

/**
 * Created by y2k on 10/12/15.
 */
public class ImageRequest extends BaseImageRequest<Bitmap> {

    @Override
    protected Bitmap decode(File path) {
        return BitmapFactory.decodeFile(path.getAbsolutePath());
    }
}

File: android/src/main/kotlin/y2k/joyreactor/platform/ImageRequest.kt
package y2k.joyreactor.platform

import android.graphics.Bitmap
import android.graphics.BitmapFactory
import y2k.joyreactor.common.images.BaseImageRequest

import java.io.File

/**
 * Created by y2k on 10/12/15.
 */
class ImageRequest : BaseImageRequest<Bitmap>() {

    override fun decode(path: File): Bitmap = BitmapFactory.decodeFile(path.absolutePath)
}